abb_irb120_support/urdf/irb120_macro.xacro

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+<xacro:macro name="abb_irb120" params="prefix">
+	<!-- link list -->
+	<link name="${prefix}base_link">
+		<visual>
+			<origin xyz="0 0 0" rpy="0 0 0"/>
+			<geometry>
+				<mesh filename="package://abb_irb120_support/meshes/irb120/visual/base_link.STL"/>
```

Comment:
I know the solidworks to urdf exporter does this by default, but we try to keep all file extensions lower case.